assets/scss/6-components/text-input/_text-input.scss

```diff
@@ -0,0 +1,20 @@
+// Text input (c-text-input)
+//
+// Text form inputs. Can be used with type="text", type="email", etc. Add .is-valid if the value is not valid.
+//
+// Markup: 6-components/text-input/text-input.html
+//
+// Styleguide 6.1.5
+.c-text-input {
+  -webkit-appearance: none;
+  border-color: $color-gray-light;
+  border-radius: 0;
+  border-style: solid;
+  border-width: 1px;
+  line-height: $line-height-b;
+  padding: $size-xxs;
+
+  &.is-invalid {
+    border-color: $color-error;
+  }
+}
```

Comment:
How would you feel about a more deliberate focus state? I've always thought relying on the browser for that was fine, but in my testing I'm noticing when the border-color is set in FF, the blue outline isn't honored on focus. Same deal with Edge. I'm not suggesting anything drastic, just a lil' something for tabbing along through a form.

Reply:
@ashley-hebler I like the idea. Do you think the focus style should be a border color, or something else? Methinks maybe something else because I'd still want users to see a red border around an input if it's invalid -- even if they did reach it by tabbing.

Reply:
@AndrewGibson27 right obviously the error border is _most_ important in this scenario, so perhaps box-shadow? Speaking of bulma, I kinda like how they handle both focus and colored borders https://bulma.io/documentation/form/input/#colors. Bootstrap also has a nice shadow application https://getbootstrap.com/docs/4.3/components/forms/

Reply:
@ashley-hebler I added a box shadow that differs based on validity. Check it out! I'd like to run the styles by art too.

Reply:
@AndrewGibson27 that looks so good! I love that it's _our_ blue. Yeah see what they say, but to me it's 💯